config/locales/en.yml

```diff
@@ -72,20 +72,52 @@ en:
       custom_select_error: Select the best way to describe your business
     product_details:
       title: Tell us about each product you're offering
+      hint: Give details for one product at a time.
```

Comment:
We don't need this one here it seems

Reply:
This is included in the new designs (see 1.1)

Reply:
I was looking at 1.3

Reply:
Ah sorry, I'll fix any issues in a followup PR